cmd/tempo-serverless/lambda/main.go

```diff
@@ -0,0 +1,104 @@
+package main
```

Comment:
nit: lets add the usual housekeeping stuff for this directory
- a \`.gitignore\` for the vendor folder
- go mod vendor/clean

Reply:
i'm torn on whether to include the vendor folder. it would be so much repeated and copied code from the original codebase sitting at both:

`./cmd/tempo-serverless/cloud-functions/vendor`
`./cmd/tempo-serverless/lambda/vendor`

i honestly think i'd be more inclined to remove the vendor folder from our main repo then add it here. the agent removed theirs here:

https://github.com/grafana/agent/pull/571